src/app/ui/button_set.h

```diff
@@ -27,6 +27,8 @@ namespace app {
       os::Surface* iconSurface() const override { return m_icon ? m_icon->bitmap(0) : nullptr; }
       skin::SkinPartPtr icon() const { return m_icon; }
       ButtonSet* buttonSet();
+
+      obs::signal<void(Item*)> Click;
```

Comment:
Is this really needed? Could we use the ButtonSet ItemChange signal? Probably it's too complex, if that is the case maybe we can just implement/derive a ButtonSet Item for the tabs widget specifically (using `ButtonSet::addItem(Item*, ...)`)

Reply:
I've considered using the ItemChange signal, but I couldn't figure out how to use it without creating a new signal for each item. Then, since I had to create a new signal anyway, I decided to connect it directly via  Dialog_connect_signal. Sure, no problem, I will create a derived class.